pages/1_📄スコア表.py

```diff
@@ -58,18 +55,42 @@
 st.title("📄スコア表")
 
 
+# select date
+date_options = df_yomma_score[DATE].drop_duplicates().sort_values(ascending=False)
+selected_date = st.selectbox("日付選択", date_options)
+## retrive scores of selected date
+df_score_table_date = df_score_table_yomma[df_score_table_yomma[DATE] == selected_date]
+## add No
+df_score_table_date[NO] = pd.RangeIndex(start=1, stop=len(df_score_table_date) + 1)
+## retrive players with any score
+columns_with_any_data = df_score_table_date.columns[
+    df_score_table_date.notna().any()
+].tolist()
+users_to_show_default = [
+    player for player in columns_with_any_data if player in user_list
+]
+
+
 # select user
-users_to_show = st.multiselect("表示する人を選択", options=user_list, default=user_list)
+users_to_show = st.multiselect(
+    "表示する人を選択", options=user_list, default=users_to_show_default
+)
 
 
-# select date
-date_options = df_yomma[DATE].drop_duplicates().sort_values(ascending=False)
-selected_date = st.selectbox("日付選択", date_options)
+# display place
+place = df_score_table_date.iloc[0][PLACE]
+st.text(f"場所：{place}")
```

Comment:
[imo] 同日に2か所で打つことはほぼないとは思うけど、念のためこんな感じはどうかな？

```suggestion
place = df_score_table_date[PLACE].unique()
st.text(f"場所：{", ".join(place)}")
```